### PR TITLE
Show thumbnails in the texture selection table.

### PR DESCRIPTION
**Pull request by pmuetschard**

### CODE REVIEW

**Comment by pmuetschard** [comment]
ping?

File: gapis/service/path/path.proto [hunk 1]
```diff
@@ -249,6 +249,6 @@ message Thumbnail {
     image.Format desired_format = 3;
 
     oneof object {
-        Command command = 4;
+        ResourceData resource = 4;
```
Comment (ben-clayton) [inline gapis/service/path/path.proto]: Is this supposed to be here?

Comment (pmuetschard) [inline gapis/service/path/path.proto]: Yes. It's even in its own commit. There is no atom that implements the Thumbnailer interface.

Comment (ben-clayton) [inline gapis/service/path/path.proto]: Sorry, that's what I get for reviewing while being in a meeting (with you).

**Pull request merged**